src/OrbitGl/GpuTrack.cpp

```diff
@@ -172,15 +172,15 @@ float GpuTrack::GetYFromTimer(const TimerInfo& timer_info) const {
   CHECK(timer_info.type() == TimerInfo::kGpuActivity ||
         timer_info.type() == TimerInfo::kGpuCommandBuffer);
 
+  float gap_space = adjusted_depth * layout_->GetSpaceBetweenGpuDepths();
```

Comment:
Add a comment why this is done here and not later (as before).

Reply:
Done. Thanks for the review.